database/migrations/2021_25_01_0127_create_isis_adjacencies_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateIsisAdjacenciesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('isis_adjacencies', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('device_id');
+            $table->integer('port_id');
```

Comment:
missing index on both

Reply:
$table->index(['device_id', 'port_id']) should do this?